coral/src/app/components/PreviewBanner.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import PreviewBanner from "src/app/components/PreviewBanner";
+
+describe("PreviewBanner.tsx", () => {
+  beforeAll(() => {
+    render(<PreviewBanner linkTarget={"/browseTopics"} />);
+  });
+  afterAll(() => {
+    cleanup();
+  });
+  it("has copy explaining that user is viewing preview UI", () => {
```

Comment:
Minor possible improvement: 

The test could look like: 

```
  it("has an accessible copy explaining that user is viewing preview UI", () => {
    const region = screen.getByRole("region", { name: "Preview disclaimer" });

    expect(region).toBeVisible();
    expect(region).toHaveTextContent(
      /You are viewing a preview of the redesigned user interface./i
    );
  });
```

That way, we would still ensure that the right copy is there, but also confirm that we have the accessible region.

Reply:
Excellent! I have not yet been able twist my brains to be as accessibility focused. These comments will help me to do so.